import createError from "http-errors";

// ===========================================================================================
// Create new User
// ===========================================================================================

export const createUser = async (req, res, next) => {
  try {
  } catch (error) {}
};

// ===========================================================================================
// Login User
// ===========================================================================================

export const loginUser = async (req, res, next) => {
  try {
  } catch (error) {}
};

// ===========================================================================================
// Update User
// ===========================================================================================

export const updateUser = async (req, res, next) => {
  try {
  } catch (error) {}
};

// ===========================================================================================
// Logout User
// ===========================================================================================

export const logoutUser = async (req, res, next) => {
    try {
    } catch (error) {}
  };
